### PR TITLE
[ACS-5166] Changes to Digital workspace new UI for better experience

### PR DESCRIPTION
**Pull request by arohilaGL**
**Please check if the PR fulfills these requirements**

> - [x] The commit message follows our [guidelines](https://github.com/Alfresco/alfresco-ng2-components/wiki/Commit-format)
> - [ ] Tests for the changes have been added (for bug fixes / features)
> - [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x")

> - [ ] Bugfix
> - [x] Feature
> - [ ] Code style update (formatting, local variables)
> - [ ] Refactoring (no functional changes, no api changes)
> - [ ] Build related changes
> - [ ] Documentation
> - [ ] Other... Please describe:

**What is the current behaviour?** (You can also link to an open issue here)

**What is the new behaviour?**

Design changes to new UI layout

**Does this PR introduce a breaking change?** (check one with "x")

> - [ ] Yes
> - [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:
https://alfresco.atlassian.net/browse/ACS-5166

### CODE REVIEW

File: lib/core/src/lib/assets/images/library_add.svg
Comment (DenysVuika) [inline lib/core/src/lib/assets/images/library_add.svg:1]: Already present in the material icons: https://fonts.google.com/icons?selected=Material+Symbols+Outlined:library_add:FILL@0;wght@400;GRAD@0;opsz@48

File: lib/core/src/lib/assets/images/icon-notification-outline-default.svg
Comment (DenysVuika) [inline lib/core/src/lib/assets/images/icon-notification-outline-default.svg:1]: Already present in the material icons: https://fonts.google.com/icons?selected=Material+Symbols+Outlined:notifications:FILL@0;wght@400;GRAD@0;opsz@48&icon.query=bell

File: lib/core/src/lib/notifications/components/notification-history.component.ts [hunk 1]
```diff
@@ -55,10 +57,22 @@ export class NotificationHistoryComponent implements OnDestroy, OnInit, AfterVie
...
+    };
+
     constructor(
         private notificationService: NotificationService,
         public storageService: StorageService,
-        public cd: ChangeDetectorRef) {
+        public cd: ChangeDetectorRef,
+        private matIconRegistry: MatIconRegistry,
+        private sanitizer: DomSanitizer) {
+            Object.keys(this.icons).forEach((key) => {
```
Comment (DenysVuika) [inline lib/core/src/lib/notifications/components/notification-history.component.ts:71]: this is incorrect, see "adf-icon" docs on how to use the icon service.

File: lib/core/src/lib/notifications/components/notification-history.component.html [hunk 2]
```diff
@@ -6,10 +6,8 @@
             class="adf-notification-history-menu_button"
             id="adf-notification-history-open-button"
             (menuOpened)="onMenuOpened()">
-        <mat-icon matBadge="&#8288;"
-                  [matBadgeHidden]="!notifications.length"
-                  matBadgeColor="accent"
-                  matBadgeSize="small">notifications</mat-icon>
+            <mat-icon *ngIf="notifications.length" svgIcon="notification:pending_notifications"></mat-icon>
```
Comment (DenysVuika) [inline lib/core/src/lib/notifications/components/notification-history.component.html:9]: should be `<adf-icon>` if you register custom images

File: lib/core/src/lib/assets/images/create_new_folder.svg
Comment (DenysVuika) [inline lib/core/src/lib/assets/images/create_new_folder.svg:1]: Already present in the material icons https://fonts.google.com/icons?selected=Material+Symbols+Outlined:create_new_folder:FILL@0;wght@400;GRAD@0;opsz@48&icon.query=folder

File: lib/core/src/lib/assets/images/draft.svg
Comment (DenysVuika) [inline lib/core/src/lib/assets/images/draft.svg:1]: Already present https://fonts.google.com/icons?selected=Material+Symbols+Outlined:draft:FILL@0;wght@400;GRAD@0;opsz@48&icon.query=document

**Review by DenysVuika — CHANGES_REQUESTED** [review]
Please use Material Icons where available already

File: lib/core/src/lib/common/services/thumbnail.service.ts [hunk 3]
```diff
@@ -158,7 +158,11 @@ export class ThumbnailService {
         'save-as': './assets/images/save-as.svg',
         save: './assets/images/save.svg',
         task: './assets/images/task.svg',
-        'multipart/related': './assets/images/ft_ic_website.svg'
+        'multipart/related': './assets/images/ft_ic_website.svg',
+        side_nav_collapse: './assets/images/icon-expand-navigation.svg',
```
Comment (MichalKinas) [inline lib/core/src/lib/common/services/thumbnail.service.ts:162]: Please change `_` to `-` to comply with already created icons

Comment (arohilaGL) [reply lib/core/src/lib/common/services/thumbnail.service.ts]: changed

File: lib/core/src/lib/assets/images/draft.svg
Comment (arohilaGL) [reply lib/core/src/lib/assets/images/draft.svg:1]: done

File: lib/core/src/lib/assets/images/create_new_folder.svg
Comment (arohilaGL) [reply lib/core/src/lib/assets/images/create_new_folder.svg:1]: done

File: lib/core/src/lib/notifications/components/notification-history.component.html (hunk 2 above)
Comment (arohilaGL) [reply lib/core/src/lib/notifications/components/notification-history.component.html]: made changes to the existing code, didnt have to use adf-icon

File: lib/core/src/lib/assets/images/icon-notification-outline-default.svg
Comment (arohilaGL) [reply lib/core/src/lib/assets/images/icon-notification-outline-default.svg:1]: done

File: lib/core/src/lib/assets/images/library_add.svg
Comment (arohilaGL) [reply lib/core/src/lib/assets/images/library_add.svg:1]: done

File: lib/core/src/lib/assets/images/icon-collapse-navigation.svg
Comment (DenysVuika) [inline lib/core/src/lib/assets/images/icon-collapse-navigation.svg:1]: can we use existing material icon here? https://fonts.google.com/icons?selected=Material+Symbols+Outlined:keyboard_double_arrow_left:FILL@0;wght@400;GRAD@0;opsz@48&icon.query=arrow

**Review by DenysVuika — APPROVED** [review]

File: lib/core/src/lib/icon/icon.component.ts [hunk 4]
```diff
@@ -48,6 +49,8 @@ export class IconComponent {
     set value(value: string) {
         this._value = value || 'settings';
         this._isCustom = this._value.includes(':');
+        this.isOutlined = this._value.includes('_outline');
+        this._value = this._value.replace('_outline','');
```
Comment (DenysVuika) [inline lib/core/src/lib/icon/icon.component.ts:53]: this is hacking of the old material icons font, and is introducing IDs that are invalid for the new Material Symbols font. this is also not going to work for cases like

```
<mat-icon class="adf-error-icon">error_outline</mat-icon>
```

**Review by DenysVuika — CHANGES_REQUESTED** [review]

**Review by DenysVuika — CHANGES_REQUESTED** [review]

**Comment by DenysVuika** [comment]
@arohilaGL you based your PR on the wrong library, you should have been using Material Icons
Screenshot 2023-06-19 at 16 28 43

**Comment by DenysVuika** [comment]
closing in favor other PRs, also the icons for mime types needed a replacement, not a separate copies, see https://github.com/Alfresco/alfresco-ng2-components/pull/8683

**Pull request closed without merging**